console/cdvdcontrol/cdvdcontrol.cpp

```diff
@@ -104,148 +102,132 @@
+      if (!plextor_get_securec_state(dev))
         dev->ven_features |= PX_SECUREC;
-      }
     }
     if (dev->wr_capabilities & DEVICE_DVD) {
-      if (plextor_get_varirec(dev, VARIREC_DVD) == 0) {
+      if (!plextor_get_varirec(dev, VARIREC_DVD))
         dev->ven_features |= PX_VARIREC_DVD;
-      }
-      if (plextor_get_bitset(dev, PLEX_BITSET_R) == 0) {
+      if (!plextor_get_bitset(dev, PLEX_BITSET_R))
         dev->ven_features |= PX_BITSET_R;
-      }
-      if (plextor_get_bitset(dev, PLEX_BITSET_RDL) == 0) {
+      if (!plextor_get_bitset(dev, PLEX_BITSET_RDL))
         dev->ven_features |= PX_BITSET_RDL;
-      }
-      if (plextor_get_autostrategy(dev) == 0) {
+      if (!plextor_get_autostrategy(dev))
         dev->ven_features |= PX_ASTRATEGY;
-      }
-      if (plextor_get_testwrite_dvdplus(dev) == 0) {
+      if (!plextor_get_testwrite_dvdplus(dev))
         dev->ven_features |= PX_SIMUL_PLUS;
-      }
     }
 #warning "PlexEraser DETECTION. Just assume PX755/760 and Premium-II"
     // if ((dev->dev_ID == PLEXTOR_755) || (dev->dev_ID == PLEXTOR_760) ||
     // (dev->dev_ID == PLEXTOR_PREMIUM2))
-    if (isPlextorLockPresent(dev) != 0) {
+    if (isPlextorLockPresent(dev))
       dev->ven_features |= PX_ERASER;
-    }
-    if (yamaha_check_amqr(dev) == 0) {
+    if (!yamaha_check_amqr(dev))
       dev->ven_features |= YMH_AMQR;
-    }
-  } else if (isYamaha(dev) != 0) {
-    if (yamaha_check_amqr(dev) == 0) {
+  } else if (isYamaha(dev)) {
+    if (!yamaha_check_amqr(dev))
       dev->ven_features |= YMH_AMQR;
-    }
-    if (yamaha_check_forcespeed(dev) == 0) {
+    if (!yamaha_check_forcespeed(dev))
       dev->ven_features |= YMH_FORCESPEED;
-    }
-    if (yamaha_f1_get_tattoo(dev) == 0) {
+    if (!yamaha_f1_get_tattoo(dev))
       dev->ven_features |= YMH_TATTOO;
-    }
-  } else if (isPioneer(dev) != 0) {
-    if (pioneer_get_quiet(dev) == 0) {
+  } else if (isPioneer(dev)) {
+    if (!pioneer_get_quiet(dev))
       dev->ven_features |= PIO_QUIET;
-    }
   }
 
-  //  printf("Trying opcode E9 modes...\n");
-  //  for (int i=0; i<256; i++) {if (!plextor_get_mode(dev,i)) printf(" MODE
-  //  0x%02X\n",i);}
+  //	printf("Trying opcode E9 modes...\n");
+  //	for (int i=0; i<256; i++) {if (!plextor_get_mode(dev,i)) printf(" MODE
+  // 0x%02X\n",i);}
 
-  //  printf("Trying opcode ED modes...\n");
-  //  for (int i=0; i<256; i++) {if (!plextor_get_mode2(dev,i)) printf(" MODE
-  //  0x%02X\n",i);}
+  //	printf("Trying opcode ED modes...\n");
+  //	for (int i=0; i<256; i++) {if (!plextor_get_mode2(dev,i)) printf(" MODE
+  // 0x%02X\n",i);}
 
-  if ((flags & FL_SUPPORTED) != 0U) {
-    //      printf("____________________________\n");
-    cout << "\n** Supported features:\n";
-    cout << "AudioMaster Q.R.    : "
-         << (((dev->ven_features & YMH_AMQR) != 0U) ? "YES" : "-") << "\n";
+  if (flags & FL_SUPPORTED) {
+    //		printf("____________________________\n");
+    printf("\n** Supported features:\n");
```

Comment:
Codacy found an issue: [Clarify calculation precedence for '&' and '?'.](https://app.codacy.com/manual/jeremiahwilson777/qpxtool/pullRequest?prid=6086205)